### PR TITLE
reduce OSX build time so that Travis is happy

### PR DESCRIPTION
**Pull request by emmanuelle**
Solves #5066 (or tries to).

### CODE REVIEW

File: tools/travis/osx_install.sh [hunk 1]
```diff
@@ -3,25 +3,12 @@ set -ex
 
 brew update
 brew install ccache
-brew tap homebrew/homebrew-cask
```
Comment (emmanuelle) [inline tools/travis/osx_install.sh:6]: probably we don't need the two lines above either?

**Comment by emmanuelle** [comment]
This is now ready for review. I'm glad Travis is green again. The biggest change of this PR is to remove some unnecessary installations is `osx_install.sh`, like latex etc. (I guess this was used when the doc was built on osx but this is not the case now).

**Review by rfezzani — APPROVED** [review]
:tada: Thank you @emmanuelle!

**Review by jni — APPROVED** [review]
Yesss! 😄 🎉 🥳 Thank you so much @emmanuelle!

**Pull request merged**